components/services/bat_ads/bat_ads_impl.cc

```diff
@@ -49,18 +46,28 @@ void BatAdsImpl::Initialize(
   auto* holder = new CallbackHolder<InitializeCallback>(AsWeakPtr(),
       std::move(callback));
 
-  ads_->Initialize(std::bind(BatAdsImpl::OnInitialize, holder, _1));
+  ads_->Initialize(
+      std::bind(BatAdsImpl::OnInitialize, holder, std::placeholders::_1));
```

Comment:
`std::bind` is banned in chromium. I know you are not introducing its use here, but would it be possible to have `base::OnceCallback` replacing the few occurrences of std::bind in this change?

https://chromium.googlesource.com/chromium/src/+/main/styleguide/c++/c++-features.md#Bind-Operations-banned

Reply:
we are fixing it in a separate ticket https://github.com/brave/brave-browser/issues/20940